build.sbt

```diff
@@ -221,6 +221,15 @@ assemblyMergeStrategy in assembly := {
     oldStrategy(x)
 }
 
+Project.inConfig(Test)(baseAssemblySettings)
+// sbt test:assembly generates test codes complied which is used in Python side tests.
+test in (Test, assembly) := {}
+fullClasspath in (Test, assembly) := {
+  val cp = (fullClasspath in (Test, assembly)).value
+  cp.filter(_.data.getPath.contains("test-classes"))
+}
+assemblyJarName in (Test, assembly) := s"${name.value}-test-${version.value}.jar"
```

Comment:
Python side needs a test codes (for example, MockConnection). So, I had to add this to make a jar file that only contains test codes. This can be triggered by `sbt test:assembly` and it makes a tar like spark-llap-test-1.2-2.3-3.0-SNAPSHOT.jar.